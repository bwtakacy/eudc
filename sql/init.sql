--
-- First, define the conversion.  Turn off echoing so that expected file
-- does not depend on the installer script.
--
SET client_min_messages = warning;
\set ECHO none
\i eudc.sql
DROP DATABASE IF EXISTS contrib_regression_utf8;
\set ECHO all
RESET client_min_messages;
