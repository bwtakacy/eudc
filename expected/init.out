--
-- First, define the conversion.  Turn off echoing so that expected file
-- does not depend on the installer script.
--
SET client_min_messages = warning;
\set ECHO none
RESET client_min_messages;
